### PR TITLE
test sv/runsvdir first

### PR DESCRIPTION
**Pull request by a-schaefers**

### CODE REVIEW

File: lib/init/rc.shutdown [hunk 1]
```diff
@@ -8,8 +8,10 @@ main() {
     PATH=/sbin:/bin:/usr/sbin:/usr/bin
 
     log "Waiting for services to stop..."; {
-        sv -w196 force-stop /var/service/*
-        sv exit /var/service/*
+        if command -v sv && [ "$(pgrep runsvdir)" ]; then
```
Comment (dylanaraps) [inline lib/init/rc.shutdown]: `command -v sv >/dev/null 2>&1 && pgrep runsvdir >/dev/null 2>&1 && {`

Then replace `fi` with `}`.

(Also, see my IRC messages)

Comment (dylanaraps) [reply lib/init/rc.shutdown]: An alternative which is simpler is to just add `>/dev/null 2>&1` to both `sv` calls. If the command isn't installed nothing happens and if it is installed but not in use, nothing will happen also.

Comment (a-schaefers) [reply lib/init/rc.shutdown]: Simpler can be better.

**Comment by a-schaefers** [comment]
mmk

File: lib/init/rc.shutdown [hunk 2]
```diff
@@ -8,10 +8,8 @@ main() {
     PATH=/sbin:/bin:/usr/sbin:/usr/bin
 
     log "Waiting for services to stop..."; {
-        if command -v sv && [ "$(pgrep runsvdir)" ]; then
-            sv -w196 force-stop /var/service/*
-            sv exit /var/service/*
-        fi
+        sv -w196 force-stop /var/service/* >/dev/null 2>&1
+        sv exit /var/service/* >/dev/null 2>&1
     }
```
Comment (dylanaraps) [inline lib/init/rc.shutdown]: Even simpler.

`} >/dev/null 2>&1`

Basically, make it affect the whole block at once!

Comment (a-schaefers) [reply lib/init/rc.shutdown]: little by little I will get there

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**